src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './Footer.css';
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <section className="footer-about-us">
+        <h3>About us</h3>
+        <ul className="list-style-none footer-list">
+          <li className="footer-list-items">
+            <a href="#">Our Story</a>
+          </li>
+          <li className="footer-list-items">
+            <a href="#">Blogs</a>
+          </li>
+          <li className="footer-list-items">
+            <a href="#">Career</a>
```

Comment:
I believe a tag is only temporary and you will later switch it with React Router Link tag.

Reply:
yup, thanks for suggestions.